replica/diff_disk.go

```diff
@@ -42,7 +43,7 @@ func (d *diffDisk) WriteAt(buf []byte, offset int64) (int, error) {
 	startSector := offset / d.sectorSize
 	sectors := int64(len(buf)) / d.sectorSize
 
-	c, err := d.files[target].WriteAt(buf, offset)
+	c, err := fio.WriteAt(d.files[target], buf, offset)
```

Comment:
Can we avoid the copy and just make the original `[]byte` aligned?  The buffer is originally allocated at https://github.com/rancher/longhorn/blob/master/rpc/wire.go#L68